### PR TITLE
Add support for HTTP chunked transfer in gremlin-python

### PR DESCRIPTION
**Pull request by xiazcy**
Added support for http chunked transfer, removed websocket and replaced old serializers with graphbinaryv4. 

Tests pass in old HTTP server with bytecode, but will not pass current one until bytecode removal is completed in python and Docker testing with new http server is set up.

### CODE REVIEW

File: gremlin-python/src/main/python/gremlin_python/driver/serializer.py [hunk 1]
```diff
@@ -386,10 +129,15 @@ def deserialize_message(self, message):
 
     def read_payload(self, buffer):
         results = []
-        while buffer.readable():
+        readable = True
+        while buffer.readable():  # find method or way to access readable bytes without using buffer.getvalue()
+            # print(buffer.tell())
```
Comment (vkagamlyk) [inline gremlin-python/src/main/python/gremlin_python/driver/serializer.py:134]: nit

**Review by vkagamlyk — COMMENTED** [review]
is it possible to fix or turn off invalid tests?

**Comment by xiazcy** [comment]
> is it possible to fix or turn off invalid tests?

Yes, there is a flag to skip them. However, right now tests won't pass anyways because we are sending bytecode, and Docker isn't set up for it. I plan to remove bytecode next so we can re-enable all integration tests.

**Comment by vkagamlyk** [comment]
VOTE+1

File: gremlin-python/src/main/python/gremlin_python/driver/protocol.py [hunk 2]
```diff
@@ -203,6 +60,12 @@ def __init__(self,
         self._message_serializer = message_serializer
         self._username = username
         self._password = password
+        self._response_msg = {'status': {'code': 0,
```
Comment (kenhuuu) [inline gremlin-python/src/main/python/gremlin_python/driver/protocol.py:63]: 0 is probably not the right default for code unless you are explicitly checking for 0 as a sentinel value.

**Comment by kenhuuu** [comment]
VOTE +1

File: gremlin-python/src/main/python/gremlin_python/driver/protocol.py (hunk 2 above)
Comment (xiazcy) [reply gremlin-python/src/main/python/gremlin_python/driver/protocol.py:63]: I think I'll leave it for now, these values do get replaced in the chunking methods as soon as we read the response.

**Comment by xiazcy** [comment]
VOTE +1

**Pull request merged**